testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/single/infinispan/query/HotRodClientTestCase.java

```diff
@@ -61,49 +65,48 @@
 @RunWith(Arquillian.class)
 public class HotRodClientTestCase {
 
-    private RemoteCache<String, Object> remoteCache;
+    private RemoteCache<String, Person> remoteCache;
 
     @Deployment
     public static Archive<?> deployment() {
-        return ShrinkWrap
-                .create(WebArchive.class, HotRodClientTestCase.class.getSimpleName() + ".war")
+        return ShrinkWrap.create(WebArchive.class, HotRodClientTestCase.class.getSimpleName() + ".war")
                 .addClass(HotRodClientTestCase.class)
-                .addClass(Person.class)
-                .add(new StringAsset(Descriptors.create(ManifestDescriptor.class).attribute("Dependencies", "org.infinispan, org.infinispan.commons, org.infinispan.client.hotrod, org.infinispan.query, org.infinispan.protostream").exportAsString()), "META-INF/MANIFEST.MF")
+                .addPackage(Person.class.getPackage())
+                .addAsServiceProvider(SerializationContextInitializer.class.getName(), PersonSchema.class.getName() + "Impl")
+                .setManifest(new StringAsset(Descriptors.create(ManifestDescriptor.class).attribute("Dependencies", "org.infinispan, org.infinispan.commons, org.infinispan.client.hotrod, org.infinispan.query, org.infinispan.protostream").exportAsString()))
                 ;
     }
 
-    private static RemoteCache<String, Object> createRemoteCache() {
-        RemoteCacheManager remoteCacheManager = new RemoteCacheManager(createConfiguration(), true);
-        RemoteCache<String, Object> remoteCache = remoteCacheManager.getCache();
-        remoteCache.clear();
-        return remoteCache;
-    }
-
-    private static Configuration createConfiguration() {
+    @Before
+    public void initialize() {
+        List<GeneratedSchema> schemas = ServiceLoader.load(SerializationContextInitializer.class, this.getClass().getClassLoader()).stream().map(ServiceLoader.Provider::get).filter(GeneratedSchema.class::isInstance).map(GeneratedSchema.class::cast).collect(Collectors.toList());
+System.out.println("LOADED SCHEMAS: " + schemas.stream().map(GeneratedSchema::getProtoFileName).collect(Collectors.toList()));
```

Comment:
Unintended `System.out`.

Reply:
Whoops!  I never pushed my final changes.  Ignore the noise for now.